test/unit/org/apache/cassandra/cql3/PreparedStatementsTest.java

```diff
@@ -54,7 +54,7 @@ public static void setup() throws Exception
 
         // Currently the native server start method return before the server is fully binded to the socket, so we need
         // to wait slightly before trying to connect to it. We should fix this but in the meantime using a sleep.
-        Thread.sleep(500);
```

Comment:
Just as a matter of test hygiene it's better to spin for a longish period of time (30 seconds) rather then hard coding a sleep of 1.5 seconds that might end up being inadequate very rarely causing spurious failures.